src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -1924,43 +1931,11 @@ internal void WriteOutput(string outputString, bool includeNewLine)
             }
         }
 
-        private void WriteExceptionToHost(Exception e)
+        private void WriteExceptionToHost(RuntimeException e)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'WriteExceptionToHost' a static method.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=5036929)

Reply:
this literally has a call to `ExecuteCommandAsync` in it. why codacy